.travis.yml

```diff
@@ -1,18 +1,18 @@
 language: node_js
 
 node_js:
-  - 17
+  - 16
 
 cache:
-  # cache both npm modules and Cypress binary
+  # cache both npm modules 
   directories:
     - ~/.npm
     - ~/.cache
-  # install dependencies and check that Cypress can run
-  override:
-    - npm ci
-    - npm run cy:verify
 
 script:
+  - npm run format
```

Comment:
you don't need `npm run format` in Travis config file, because there is no point in formatting files in CI. Because then you can push your code without following formatting rules but it will always pass in CI.
1. run `npm run format` locally and push the changes.
2. remove `npm run format` from this file

Reply:
Has been fixed